blazor/datagrid/detail-template.md

```diff
@@ -328,4 +328,102 @@ By default, detail rows render in collapsed state. You can expand a detail row b
 ```
 
 > * You can expand all the rows by using `ExpandAll` method.
-> * If you want to expand all the rows at initial DataGrid rendering, then use `ExpandAll` method in [dataBound](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html) event of the DataGrid.
\ No newline at end of file
+> * If you want to expand all the rows at initial DataGrid rendering, then use `ExpandAll` method in [dataBound](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html) event of the DataGrid.
+
+## How to set grid instance for child component in detail template
+
+The Grid component has an option to get the child (custom) component details, which are rendered inside the detail template. This can be achieved by setting the instance as a dictionary object in the child component. Also, the key values should be unique.
```

Comment:
Hierarchy Grid like structure can be achieved using detail template feature of Grid. We can get instance of each Child grid by defining the @ref property with dictionary object in Detailtempalte Grid component. Also the key value should be unique.